src/network/connection.cpp

```diff
@@ -825,8 +825,9 @@ void Peer::DecUseCount()
 	delete this;
 }
 
-void Peer::RTTStatistics(float rtt, const std::string &profiler_id,
-		unsigned int num_samples) {
+void Peer::RTTStatistics(float rtt, const std::string &profiler_id)
+{
+	static const float avg_factor = 0.1f / MAX_RELIABLE_WINDOW_SIZE;
```

Comment:
seems to be constexpr const instead of just const

Reply:
`MAX_RELIABLE_WINDOW_SIZE` is a define, so const should be okay